packages/astro/README.md

```diff
@@ -21,3 +21,33 @@ npm init astro
 ```
 
 ### [Full Documentation Site →](https://docs.astro.build/)
+
+## Sponsors
+
+Astro is generously supported by the following companies and individuals.
+
+### Platinum Sponsors
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center"><a href="https://www.netlify.com/" target="_blank"><img width="147" height="40" src="./.github/assets/netlify.svg" alt="Netlify" /></a></td>
+    </tr>
+  </tbody>
+</table>
+
+### Sponsors
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center"><a href="https://sentry.io" target="_blank"><img width="147" height="40" src="./.github/assets/sentry.svg" alt="Sentry" /></a></td>
+    </tr>
+  </tbody>
+</table>
+
+### Donate
+
+<a href="https://opencollective.com/astrodotbuild/donate" target="_blank">
```

Comment:
this button + header takes up a lot of real estate. What about just moving this link into the larger section header:

"You can sponsor Astro's development on [Open Collective](LINK). Astro is generously supported by the following companies and individuals."

Reply:
Cool. I agree I’m not liking the button either. Just didn’t want people to scroll past it. But on the other hand it’s probably not like anyone sees that button and decides to donate all of a sudden.